modules/experimental-layers/src/tile-layer/utils/tile-cache.js

```diff
@@ -30,48 +36,93 @@ export default class TileCache {
    * @param {*} onUpdate
    */
   update(viewport, onUpdate) {
-    const {cache, size, getTileData} = this;
-    const tiles = getTileIndices(viewport).map(({x, y, z}) => {
-      let tile = this._find(x, y, z);
+    const {_cache, _getTileData, _maxSize} = this;
+    this._markOldTiles();
+    const tileIndices = getTileIndices(viewport);
+    if (!tileIndices || tileIndices.length === 0) {
+      onUpdate(tileIndices);
+      return;
+    } else if (
+      this._minZoom &&
+      this._minZoom === parseInt(this._minZoom, 10) &&
```

Comment:
What does this do?

Reply:
This checks whether `this._minZoom` is an integer

Reply:
You can normalize minZoom and maxZoom in the constructor once and for all.

Reply:
Yep, moved this to the constructor already.